ext/intl/dateformat/dateformat.stub.php

```diff
@@ -154,10 +154,11 @@ public static function formatObject($datetime, $format = null, ?string $locale =
 
     /**
      * @param int $offset
+     * @param bool $updateCalendar
      * @tentative-return-type
      * @alias datefmt_parse
      */
-    public function parse(string $string, &$offset = null): int|float|false {}
+    public function parse(string $string, &$offset = null, bool $updateCalendar = false): int|float|false {}
```

Comment:
I don't want to bikeshed this much, but I'd still want to throw one more idea to think about:

```
public function parseToCalendar(string $string, &$offset = null): void {}
```

Maybe a signature like this would convey the message better that it is intended to modify the internal state. Let's wait for someone else's review and go with the signature which has more support :) I'm also fine with yours even though it's not my preference

Reply:
Oh, and I almost forgot: having a completely new method would also minimize the BC break, since `IntlDateformatter::parse()` is not `final`, so please may override it.

Reply:
valid points, thanks. Will update later.